snmp/datadog_checks/snmp/data/profiles/idrac.yaml

```diff
@@ -104,3 +104,197 @@ metrics:
+      OID: 1.3.6.1.4.1.674.10892.5.4.1100.90
+      name: networkDeviceTable
+    symbols:
+      - OID: 1.3.6.1.4.1.674.10892.5.4.1100.90.1.3
+        name: networkDeviceStatus
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.674.10892.5.4.1100.90.1.1
+          name: networkDeviceChassisIndex
+        tag: chassis_index
+      - column:
+          OID: 1.3.6.1.4.1.674.10892.5.4.1100.90.1.16
+          name: networkDevicePermanentMACAddress
+        tag: mac_addr
+      - column:
+          OID: 1.3.6.1.4.1.674.10892.5.4.1100.90.1.30
+          name: networkDeviceFQDD
+        tag: device_fqdn
+  - MIB: IDRAC-MIB-SMIv2
+    table:
+      OID: 1.3.6.1.4.1.674.10892.5.4.300.50
+      name: systemBIOSTable
+    symbols:
+      - OID: 1.3.6.1.4.1.674.10892.5.4.300.50.1.5
+        name: systemBIOSStatus
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.674.10892.5.4.300.50.1.1
+          name: systemBIOSchassisIndex
+        tag: chassis_index
+  - MIB: IDRAC-MIB-SMIv2
+    table:
+      OID: 1.3.6.1.4.1.674.10892.5.4.600.30
+      name: amperageProbeTable
+    symbols:
+      - OID: 1.3.6.1.4.1.674.10892.5.4.600.30.1.5
+        name: amperageProbeStatus
+      - OID: 1.3.6.1.4.1.674.10892.5.4.600.30.1.6
+        name: amperageProbeReading
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.674.10892.5.4.600.30.1.7
+          name: amperageProbeType
+        tag: probe_type
+      - column:
+          OID: 1.3.6.1.4.1.674.10892.5.4.600.30.1.1
+          name: amperageProbechassisIndex
+        tag: chassis_index
+  - MIB: IDRAC-MIB-SMIv2
+    table:
+      OID: 1.3.6.1.4.1.674.10892.5.4.600.20
+      name: voltageProbeTable
+    symbols:
+      - OID: 1.3.6.1.4.1.674.10892.5.4.600.20.1.5
+        name: voltageProbeStatus
+      - OID: 1.3.6.1.4.1.674.10892.5.4.600.20.1.6
+        name: voltageProbeReading
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.674.10892.5.4.600.20.1.1
+          name: voltageProbechassisIndex
+        tag: chassis_index
+      - column:
+          OID: 1.3.6.1.4.1.674.10892.5.4.600.20.1.7
+          name: voltageProbeType
+        tag: probe_type
+  - MIB: IDRAC-MIB-SMIv2
+    table:
+      OID: 1.3.6.1.4.1.674.10892.5.4.1100.50
+      name: memoryDeviceTable
+    symbols:
+      - OID: 1.3.6.1.4.1.674.10892.5.4.1100.50.1.5
+        name: memoryDeviceStatus
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.674.10892.5.4.1100.50.1.7
+          name: memoryDeviceType
+        tag: device_type
+      - column:
+          OID: 1.3.6.1.4.1.674.10892.5.4.1100.50.1.1
+          name: memoryDevicechassisIndex
+        tag: chassis_index
+
+  - MIB: DELL-RAC-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.674.10892.2.3.1.9
+      name: drsCMCCurrStatus
+ #   metric_tags:
+ #     - MIB: DELL-RAC-MIB
+ #       column:
+ #         OID: 1.3.6.1.4.1.674.10892.2.4.1.1.1
+ #         name: drsChassisIndex
+ #       table: drsCMCPowerTableEntry
+ #       tag: chassis_index
+
```

Comment:
These tags were not parsing correctly

Reply:
Yeah this doesn't make sense. You can't tag a single value with a table value. We should remove those.

Reply:
Sounds good! I removed them